api-simple/next-steps.md

```diff
@@ -0,0 +1,61 @@
+
+# Next Steps
```

Comment:
@savannahostrowski @gkulin Made some changes based on feedback and added troubleshooting, let me know how this looks for you